### ISSUE TITLE
neutral faction select error on Classic Era

### ISSUE THREAD

**Issue by rodneycheung** [GoldpawsStuff/BigInputBox#5]
All addons disabled. Latest version from Curse

```
Message: [string "BigInputBoxEditBoxLanguageButton:OnLoad"]:72: Attempt to register unknown event "NEUTRAL_FACTION_SELECT_RESULT"
Time: Sun Oct 17 17:07:18 2021
Count: 1
Stack: [string "BigInputBoxEditBoxLanguageButton:OnLoad"]:72: Attempt to register unknown event "NEUTRAL_FACTION_SELECT_RESULT"
[string "=[C]"]: in function `RegisterEvent'
[string "*:OnLoad"]:72: in function <[string "*:OnLoad"]:1>

Locals: (*temporary) = BigInputBoxEditBoxLanguageButton {
 0 = <userdata>
 LoadLanguages = <function> defined *:OnLoad:15
 Update = <function> defined *:OnLoad:46
 Language = <unnamed> {
 }
 currentLanguageIndex = 1
}
(*temporary) = "NEUTRAL_FACTION_SELECT_RESULT"
```

**Comment by Goldpaw** [comment]
Ah, yes. I can see the problem without even checking my code here. 

That event fires when a pandaren chooses faction at the end of the intro questline. An event that didn't exist until MoP, and isn't included in the classics. I'll get it fixed asap! :o

**Comment by Goldpaw** [comment]
Fixed in 1.0.16-Release!
